_data/simulations/moose_psu_1d_STA/meta.yaml

```diff
@@ -0,0 +1,58 @@
+---
+  benchmark:
+    id: 1d
+    version: 1
+
+  metadata:
+    summary: Metadata for the Benchmark1 problems
+    timestamp: 02/20/2017
+    author: PC. Simon
+    email: pjs5523@psu.edu
+    timestamp: "Tue Feb  7 10:36:59 2017 -EST"
+    software:
+      name: moose
+      version: git commit 2593a61 on 2017-01-18
+    hardware:
+      computing plateform: Red Hat Entreprise Linux 6 (Cluster)
```

Comment:
computing plateform is not a defined key word, see https://travis-ci.org/usnistgov/chimad-phase-field/builds/203624200#L891